### ISSUE TITLE
Request feature: supporting larger output files

### ISSUE THREAD

**Issue by jywu20** [MineralsCloud/QuantumESPRESSOParser.jl#97]
**Is your feature request related to a problem? Please describe.**
When I try to parse a `bands.out` file containing about 10000 lines, I get this error: 
```
ERROR: PCRE.exec error: JIT stack limit reached
Stacktrace:
 [1] error(s::String)
   @ Base .\error.jl:33
 [2] exec(re::Ptr{Nothing}, subject::String, offset::Int64, options::UInt32, match_data::Ptr{Nothing})
   @ Base.PCRE .\pcre.jl:197
 [3] exec_r_data
   @ .\pcre.jl:210 [inlined]
 [4] match(re::Regex, str::String, idx::Int64, add_opts::UInt32)
   @ Base .\regex.jl:371
 [5] match
   @ .\regex.jl:369 [inlined]
 [6] match
   @ .\regex.jl:388 [inlined]
```
The error is thrown at [this line](https://github.com/MineralsCloud/QuantumESPRESSOParser.jl/blob/748eea896cbb8ebd92cce78599f91578c3cb80a4/src/Outputs/PWscf.jl#L285). 

**Describe the solution you'd like**
Appropriate pre-processing to avoid blowing up the regex module, or maybe increasing the JIT stack as is suggested by the error message?

**Comment by singularitti** [comment]
Hi @jywu20, I have never thought a 10000 lines of data would make it out of memory. I am not familiar with how PCRE works but it seems that it is a problem of PCRE and not unique to our code or Julia:

1. [pcre error -27 (JIT stack limit) on long regex string](https://github.com/JuliaLang/julia/issues/8278)
2. [PCRE JIT stack size limit](https://externals.io/message/87245)
3. [RegEx not working for long pattern PCRE's JIT compiler stack limit - PHP7](https://stackoverflow.com/questions/34849485/regex-not-working-for-long-pattern-pcres-jit-compiler-stack-limit-php7)

**Comment by jywu20** [comment]
Yeah, I have checked all the three links before I submitted this issue. The location of this error is quite straightforward: it has nothing to do with the logic of the QuantumESPRESSO part. What concerns me is there seems to be no well-documented method to increase the memory allocated for PCRE. [This post](https://github.com/JuliaLang/julia/issues/8278#issuecomment-55316887) provides a `ccall` code to increase the allocation, which I tried but it didn't work.
